src/sentry/integrations/gitlab/client.py

```diff
@@ -243,12 +243,13 @@ def get_diff(self, project_id, sha):
         return self.get(path)
 
     @transaction_start("GitLabApiClient.check_file")
-    def check_file(self, project_id, path, ref):
+    def check_file(self, repo, path, ref):
         """Fetch a file for stacktrace linking
 
         See https://docs.gitlab.com/ee/api/repository_files.html#get-file-from-repository
         Path requires file path and ref
         """
         self.base_url = self.metadata["base_url"]
```

Comment:
@MeredithAnya @ceorourke I just noticed that we were setting the `base_url` inside `check_file` which could cause a side effect if we tried to use the API client again and we are expecting the original `base_url`. No need to fix in this PR but we should fix it eventually.